### PR TITLE
TSK-1957 extend TestCase with comparison for groups

### PR DESCRIPTION
**Pull request by arolfes**
https://sonarcloud.io/summary/new_code?id=arolfes_taskana&branch=TSK-1957-testcase-userinforefreshjob

---
Release Notes:

```

```
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [x] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] After integration of the PR, I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [ ] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: rest/taskana-rest-spring/src/test/java/pro/taskana/user/jobs/UserInfoRefreshJobIntTest.java [hunk 1]
```diff
@@ -71,6 +72,29 @@ void should_RefreshUserInfo_When_UserInfoRefreshJobIsExecuted() throws Exception
...
+                  .sorted(Comparator.naturalOrder())
+                  .collect(Collectors.toList());
+          // we know that our users from ldap have groups defined
+          assertThat(groupIds)
+              .isNotEmpty()
+              .hasSameSizeAs(ldapGroups)
+              .containsExactlyElementsOf(ldapGroups);
+        } catch (Exception e) {
+          throw new RuntimeException(e);
+        }
```
Comment (mustaphazorgati) [inline rest/taskana-rest-spring/src/test/java/pro/taskana/user/jobs/UserInfoRefreshJobIntTest.java:95]: remove

File: rest/taskana-rest-spring/src/test/java/pro/taskana/user/jobs/UserInfoRefreshJobIntTest.java [hunk 2]
```diff
@@ -71,6 +72,29 @@ void should_RefreshUserInfo_When_UserInfoRefreshJobIsExecuted() throws Exception
...
+      }
+      users.forEach(user -> {});
```
Comment (mustaphazorgati) [inline rest/taskana-rest-spring/src/test/java/pro/taskana/user/jobs/UserInfoRefreshJobIntTest.java:97]: remove

**Review by mustaphazorgati — CHANGES_REQUESTED** [review]

**Review by mustaphazorgati — APPROVED** [review]
